src/index.ts

```diff
@@ -136,7 +130,7 @@ export function createSyncFn<R, T extends AnyAsyncFn<R>>(
   const syncFn = startWorkerThread<R, T>(
     workerPath,
     /* istanbul ignore next */ typeof bufferSizeOrOptions === 'number'
```

Comment:
There should be no `bufferSize` argument anymore.

Reply:
This would be a breaking change, since there is a possibility of calling with three arguments, if we remove the middle one it should probably be for the timeout.
So people using the library will think they are setting a buffer (probably a high number) and are actually setting a timeout.

If that's OK for you, I will apply this change

Reply:
Yes, that's why I asked to remove the the option and I'm going to release this PR as minor in v0.x which means breaking change.

Reply:
`timeoutOrOptions`

Reply:
Change applied